packages/kit/src/runtime/server/page/index.js

```diff
@@ -98,7 +98,7 @@ export async function render_page(event, page, options, manifest, state, resolve
 		/** @type {import('./types.js').Fetched[]} */
 		const fetched = [];
 
-		if (get_option(nodes, 'ssr') === false && !state.prerendering) {
+		if (get_option(nodes, 'ssr') === false && !(state.prerendering && should_prerender_data)) {
```

Comment:
can we add a comment about when this is being triggered or what the response is for? e.g. is this the SPA fallback page or something like that?

Reply:
added a comment